docs/data/ampli/cli.md

```diff
@@ -54,7 +54,7 @@ Now that you're done setting Ampli up, you’re ready to generate your analytics
 ampli pull {source-name}
 ```
 
-The pull command takes one argument: the name of a source created in your Amplitude Data account. For example, a source might be `ios`, `android`, `web`, or `backend`. Events associated with a particular source appear in the autogenerated SDK. Those that aren't don't. This makes sure Amplitude tracks your events consistently across all sources you track them on.
+The pull command takes one argument: the name of a source created in your Amplitude Data account. For example, a source might be `ios`, `android`, `web`, or `backend`. Follow the prompt options to configure a newly created sources. Events associated with a particular source appear in the autogenerated SDK. Those that aren't don't. This makes sure Amplitude tracks your events consistently across all sources you track them on.
```

Comment:
nit: can we please update this to call out the arg is optional? I would like to educate users to run `ampli pull` without any arguments and instead rely on the prompts to guide the process.
cc @caseyamp 
```suggestion
The pull command takes one (optional) argument: the name of a source created in your Amplitude Data account. For example, a source might be `ios`, `android`, `web`, or `backend`. Follow the prompt options to configure a newly created sources. Events associated with a particular source appear in the autogenerated SDK. Those that aren't don't. This makes sure Amplitude tracks your events consistently across all sources you track them on.
```

Reply:
I think it might flow better to say "the pull command takes one optional argument", but I'm okay with this.